models/src/node/mod.rs

```diff
@@ -9,6 +9,11 @@ pub use self::error::Error as BottlerocketShadowError;
 
 use lazy_static::lazy_static;
 pub use semver::Version;
+use snafu::ResultExt;
+use std::io::Read;
+
+/// The module-wide result type.
+type Result<T> = std::result::Result<T, mod_error::Error>;
```

Comment:
Isn't this already defined here?

https://github.com/bottlerocket-os/bottlerocket-update-operator/blob/72bf1b36c003bc270daee59464d68a3fee2eaf1f/models/src/node/error.rs#L5